peachjam/models/core_document_model.py

```diff
@@ -87,6 +88,9 @@ class Locality(models.Model):
         Country, on_delete=models.PROTECT, verbose_name=_("jurisdiction")
     )
     code = models.CharField(_("code"), max_length=20, null=False)
+    entity_profile = GenericRelation(
+        "peachjam.EntityProfile", verbose_name=_("profile")
+    )
```

Comment:
interesting that this works without a migration

Reply:
yeah, my thoughts too. I think it sees that there is a GenericForeignKey on the other model.